modules/lang-painless/src/main/java/org/elasticsearch/painless/Definition.java

```diff
@@ -968,8 +957,12 @@ private void copyStruct(String struct, List<String> children) {
                 MethodKey methodKey = kvPair.getKey();
                 Method method = kvPair.getValue();
                 if (owner.methods.get(methodKey) == null) {
+                    // TODO: some of these are no longer valid or outright don't work
+                    // TODO: since classes may not come from the Painless classloader
+                    // TODO: and it was dependent on the order of the extends which
+                    // TODO: which no longer exists since this is generated automatically
                     // sanity check, look for missing covariant/generic override
-                    if (owner.clazz.isInterface() && child.clazz == Object.class) {
+                    /*if (owner.clazz.isInterface() && child.clazz == Object.class) {
```

Comment:
Is this meant to be commented out? Please remove if it is no longer relevant.

Reply:
I want to follow up in a different PR.  Added a long explanation in a TODO.  Will remove the elements after I've gone through them again.  I think some are still relevant, but the code is buggy at this time.